examples/platforms/da15000/radio.c

```diff
@@ -95,15 +95,17 @@ void da15000RadioInit(void)
 void otPlatRadioGetIeeeEui64(otInstance *aInstance, uint8_t *aIeeeEui64)
 {
     (void)aInstance;
+    uint32_t *timeStamp  = (uint32_t *) 0x7F8EA08;
+    uint32_t *testId     = timeStamp + 1;
 
     aIeeeEui64[0] = 0x80;    //80-EA-CA is for Dialog Semiconductor
     aIeeeEui64[1] = 0xEA;
     aIeeeEui64[2] = 0xCA;
-    aIeeeEui64[3] = 0x00;
-    aIeeeEui64[4] = (HARDCODED_NODE_ID >> 24) & 0xff;
-    aIeeeEui64[5] = (HARDCODED_NODE_ID >> 16) & 0xff;
-    aIeeeEui64[6] = (HARDCODED_NODE_ID >>  8) & 0xff;
-    aIeeeEui64[7] =  HARDCODED_NODE_ID        & 0xff;
+    aIeeeEui64[3] = (*testId    >>  8) & 0xff;
+    aIeeeEui64[4] = (*timeStamp >> 24) & 0xff;
+    aIeeeEui64[5] = (*timeStamp >> 16) & 0xff;
+    aIeeeEui64[6] = (*timeStamp >>  8) & 0xff;
+    aIeeeEui64[7] =  *timeStamp        & 0xff;
```

Comment:
I'm not as much worried about the uniqueness, as I am about the ephemeralness of this EUI. The Thread 1.1.1 spec refers to it as a "Factory assigned global EUI-64", to me this means that it is static.  I know that the DA15000 dosnt have a factory stamped EUI, but I feel like in this case we should try to stay as close to the spec as possible and use a static value, not an ephemeral one.  Additionally, I feel like there could be other issues caused by regenerating a EUI at runtime.

@jwhui What do you think?